example/test.js

```diff
@@ -0,0 +1,9992 @@
+// source: test.proto
+/**
+ * @fileoverview
+ * @enhanceable
+ * @suppress {missingRequire} reports error on implicit type usages.
+ * @suppress {messageConventions} JS Compiler reports an error if a variable or
+ *     field starts with 'MSG_' and isn't a translatable message.
+ * @public
+ */
+// GENERATED CODE -- DO NOT EDIT!
+/* eslint-disable */
+// @ts-nocheck
+
+import * as jspb from 'google-protobuf';
+var goog = jspb;
+var proto = {};
+
+import * as google_protobuf_descriptor_pb from 'google-protobuf/google/protobuf/descriptor.js';
+goog.exportSymbol('proto.jspb.test.BooleanFields', null, global);
```

Comment:
I don't think we want any of these exportSymbols in es6 output.

Reply:
Removed.